docs/modules/ROOT/pages/erc20-flash-mint.adoc

```diff
@@ -0,0 +1 @@
+
```

Comment:
Make use of this [docs](https://docs.openzeppelin.com/contracts/5.x/api/token/erc20#ERC20FlashMint), and make it consistent with other antora docs.